realm/realm-library/src/objectServer/java/io/realm/mongodb/App.java

```diff
@@ -519,14 +492,8 @@ public AppConfiguration getConfiguration() {
      * Swap the currently configured network transport with the provided one.
      * This should only be done if no network requests are currently running.
      */
-    void setNetworkTransport(OsJavaNetworkTransport transport) {
-        networkTransport = transport;
-    }
-
-    @KeepMember
-        // Called from JNI
-    OsJavaNetworkTransport getNetworkTransport() {
-        return networkTransport;
+    public void setNetworkTransport(OsJavaNetworkTransport transport) {
```

Comment:
Does this really need to be public? It looks like it is only used by `TestApp` which already extends `App`, so it could just access internals directly.

If not, we need to add JavaDoc and warn people about not using this as it is for internal usage.